File: Chalkable.Web/app/chlk/models/academicBenchmark/Course.js
REQUIRE('chlk.models.id.ABSubjectDocumentId');
REQUIRE('chlk.models.id.ABCourseId');

NAMESPACE('chlk.models.academicBenchmark', function(){

    /**@class chlk.models.academicBenchmark.Course*/

    CLASS('Course', [

        chlk.models.id.ABCourseId, 'id',
        String, 'description',
        chlk.models.id.ABAuthorityId, 'authorityId',
        chlk.models.id.ABDocumentId, 'documentId',
        chlk.models.id.ABSubjectDocumentId, 'subjectDocumentId',
        String, 'gradeLevel',

        function getName(){
            return this.getDescription();
        }
    ]);
});

File: Chalkable.BusinessLogic/Model/AcademicBenchmark/Course.cs
using System;

namespace Chalkable.BusinessLogic.Model.AcademicBenchmark
{
    public class Course
    {
        public Guid Id { get; set; }
        public string Description { get; set; }
        public static Course Create(AcademicBenchmarkConnector.Models.Course course)
        {
            return new Course
            {
                Id = course.Id,
                Description = course.Description
            };
        }
    }
}
